lib/stores/FileStore.js

```diff
@@ -168,6 +167,32 @@ class FileStore extends DataStore {
             });
         });
     }
+
+    binPath(file_id) {
+        return `${this.directory}/${file_id}`;
+    }
+
+    configPath(file_id) {
+        return `${this.directory}/${file_id}.info`;
+    }
+
+    getConfig(file_id) {
+        try {
+            const infoFilename = this.configPath(file_id);
+            const data = fs.readFileSync(infoFilename);
+            const file = JSON.parse(data);
+            return file;
+        }
+        catch (e) {
+            return null;
```

Comment:
This should probably throw the error to in order to debug, and return an error code to the request?

Reply:
the logic in write and getoffset expects config to be falsey if it doesn't exist.
write and getOffset currently do not handle the case when file info is missing/unreadable and caused alot of test failures.
I have added a rejection when creating the file since most failures would be there.
also now log the error.